src/name_bazaar/ui/events/reverse_registrar_events.cljs

```diff
@@ -0,0 +1,23 @@
+(ns name-bazaar.ui.events.reverse-registrar-events
+  (:require
+   [cljs.spec.alpha :as s]
+   [district0x.ui.spec-interceptors :refer [validate-first-arg]]
+   [goog.string :as gstring]
+   [name-bazaar.ui.constants :as constants :refer [default-gas-price interceptors]]
+   [re-frame.core :as re-frame :refer [reg-event-fx]]
+   ))
+
+(reg-event-fx
+ :reverse-registrar/claim-with-resolver
+ [interceptors (validate-first-arg (s/keys :req [:ens.record/addr]))]
+ (fn [{:keys [:db]} [form-data]]
+   {:dispatch [:district0x/make-transaction
+               {:name (gstring/format "Setting reverse resolver for %s" (:ens.record/addr form-data))
+                :contract-key :reverse-registrar
+                :contract-method :claim-with-resolver
+                :form-data (select-keys form-data [:ens.record/addr :public-resolver])
```

Comment:
`form-data` doesn't contain `:public-resolver` address at this point. Setting up resolver doesn't work, I tried on mainnet